packages/zoe/test/unitTests/contracts/test-throwInOfferHandler.js

```diff
@@ -31,15 +31,50 @@ test('throw in offerHandler', async t => {
 
   const throwsInOfferHandlerSeat = E(zoe).offer(throwInOfferHandlerInvitation);
 
-  await t.throwsAsync(() => E(throwsInOfferHandlerSeat).getOfferResult(), {
+  const throwsInOfferHandlerResult = E(
+    throwsInOfferHandlerSeat,
+  ).getOfferResult();
+
+  // Uncomment below to see what the user would see
+  // await throwsInOfferHandlerResult;
+
+  await t.throwsAsync(() => throwsInOfferHandlerResult, {
     message: 'error thrown in offerHandler in contract',
   });
 
   const throwsInDepositToSeatSeat = E(zoe).offer(
     throwInDepositToSeatInvitation,
   );
 
-  await t.throwsAsync(() => E(throwsInDepositToSeatSeat).getOfferResult(), {
+  const throwsInDepositToSeatResult = E(
+    throwsInDepositToSeatSeat,
+  ).getOfferResult();
+
+  // Uncomment below to see what the user would see
+  // await throwsInDepositToSeatResult;
```

Comment:
Thanks for this. I can use this as a driving example for better distributed error reporting.